EScavange.py

```diff
@@ -0,0 +1,18 @@
+import database
+import gamemaker
+
+class main():
+  user = ""
```

Comment:
Do these need to be defined as variables? I thought they were being stored in the db. If you do need to have a variable here, then just have a variable of type GameMaker/Team and no password since the password will exist in the particular instance of that user.